src/utils/bemClassNames.js

```diff
@@ -6,15 +6,21 @@ export default function bemClassNames(blockName) {
         blockName = `${DEFAULT_PREFIX}${blockName}`
     }
 
+    let elementLookup = {}
+
     return {
         b: (...modifierArgs) =>
             modifierArgs.length === 0
                 ? blockName
                 : classNames(blockName, modifierArgs),
-        e: (elementName, ...modifierArgs) =>
-            modifierArgs.length === 0
-                ? `${blockName}__${elementName}`
-                : classNames(`${blockName}__${elementName}`, modifierArgs),
+        e: (elementName, ...modifierArgs) => {
```

Comment:
I don't know how the function works, but will the modifierArgs create a lookup miss